lib/order.rb

```diff
@@ -0,0 +1,55 @@
+require "csv"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  @@valid_statuses = %i[pending paid processing shipped complete]
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
```

Comment:
Since it never changes, I would use a constant for the list of valid statuses instead of a class variable.